frontend/src/features/leases/list/LeaseListView.tsx

```diff
@@ -26,7 +29,7 @@ export const LeaseListView = () => {
     setSort,
     setCurrentPage,
     setPageSize,
-  } = useSearch(defaultFilter);
+  } = useSearch<ILeaseSearchResult, ILeaseFilter>(defaultFilter, getLeases);
```

Comment:
useSearch/useFetcher updated with generics to promote reuse.

Reply:
+1